bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+
+num_users_updated = 0
+batch_size = 10000
+
+User.where.not(school_info: nil).find_in_batches(batch_size: batch_size) do |batch|
+  ActiveRecord::Base.transaction do
+    batch.each do |user|
+    puts "Processing #{num_users_updated}...."
```

Comment:
This might end up with a lot of print statements...

Reply:
Ah yes, I will update.